state-synchronizer/src/counters.rs

```diff
@@ -26,6 +26,21 @@ pub const COMMITTED_VERSION_LABEL: &str = "committed"; // Version of latest ledg
 pub const SYNCED_VERSION_LABEL: &str = "synced"; // Version of most recent txn that was synced (even if it is not backed by an LI)
 pub const TARGET_VERSION_LABEL: &str = "target"; // Version a node is trying to catch up to
 
+// failed channel send type labels
```

Comment:
if you want to have a lot, lot of fun with type safety: https://github.com/libra/libra/blob/master/secure/net/src/lib.rs#L106-L133